### ISSUE TITLE
Duplicate readings

### ISSUE THREAD

**Issue by adatum** [electrified/asus-wmi-sensors#4]
Thank you so much for making this driver!!! I have it working on a C7HWIFI with BIOS 1103 on Fedora 28 kernel 4.19.15. 

I noticed some readings seem to be duplicated with different values: CPU Core Voltage, CPU SOC Voltage, DRAM Voltage. Is this expected?

```
$ sensors
asus-isa-0000
Adapter: ISA adapter
cpu_fan:        0 RPM

asuswmisensors-isa-0000
Adapter: ISA adapter
CPU Core Voltage:         +1.19 V  
CPU SOC Voltage:          +1.12 V  
DRAM Voltage:             +1.34 V  
VDDP Voltage:             +0.72 V  
1.8V PLL Voltage:         +1.83 V  
+12V Voltage:            +11.99 V  
+5V Voltage:              +4.99 V  
3VSB Voltage:             +3.36 V  
VBAT Voltage:             +3.27 V  
AVCC3 Voltage:            +3.33 V  
SB 1.05V Voltage:         +1.10 V  
CPU Core Voltage:         +0.76 V  
CPU SOC Voltage:          +1.14 V  
DRAM Voltage:             +1.35 V  
CPU Fan:                  478 RPM
Chassis Fan 1:            484 RPM
Chassis Fan 2:            504 RPM
Chassis Fan 3:            457 RPM
HAMP Fan:                 867 RPM
Water Pump:                 0 RPM
CPU OPT:                  469 RPM
Water Flow:                 0 RPM
AIO Pump:                   0 RPM
CPU Temperature:          +41.0°C  
CPU Socket Temperature:   +29.0°C  
Motherboard Temperature:  +24.0°C  
Chipset Temperature:      +43.0°C  
Tsensor 1 Temperature:   +216.0°C  
CPU VRM Temperature:      +33.0°C  
Water In:                +216.0°C  
Water Out:               +216.0°C  
CPU VRM Output Current:   +0.00 A  

k10temp-pci-00c3
Adapter: PCI adapter
Tdie:         +41.4°C  (high = +70.0°C)
Tctl:         +51.4°C  

```

**Comment by electrified** [comment]
Good spot, I will look at removing the duplication.

Background: the sensors are in two "banks" which you have to switch between to make readings. CPU Core Voltage, CPU SOC Voltage and DRAM Voltage are duplicated in both banks. I'm guessing this is to improve performance reading those as switching between banks is a slow operation.

**Comment by adatum** [comment]
Are both banks for those three readings supposed to have the same values, with the difference due to time delays in accessing them?

**Comment by electrified** [comment]
That is correct, the Core voltage especially fluctuates a lot on Ryzen, the difference will be due to when the reading was made.

**Comment by adatum** [comment]
I'm noticing a slight but consistent difference, typically up to 0.02V, between the two readings for DRAM Voltage and also for CPU SOC Voltage. Any idea why?

CPU Core Voltage can be much more different, but the measurement/polling frequency could explain that one.

![xsensors_diffs](https://user-images.githubusercontent.com/9773655/51491665-11e74f00-1d7d-11e9-9277-0061bdb32c67.png)

**Comment by electrified** [comment]
Thanks for posting, you made me check some Windows tools that use the Asus WMI output namely HWiNFO and SIV. My previous theory about the duplication was wrong, they are from different sources. The second set is the voltages read from the VRMs by the embedded controller!

Both of these tools also report the 2 sets of figures, but rename them for clarity

HWiNFO suffixes them (VRM)

<img width="519" alt="hwinfo" src="https://user-images.githubusercontent.com/103082/51493134-30b10a00-1dac-11e9-8f97-29b8a825abd4.PNG">

SIV suffixes them (EC)

<img width="726" alt="siv hardware status" src="https://user-images.githubusercontent.com/103082/51493152-47eff780-1dac-11e9-82f0-c19d80d2a21d.PNG">

And the raw readings in SIV with unrenamed descriptions, which is what I'm displaying:

<img width="785" alt="siv raw wmi" src="https://user-images.githubusercontent.com/103082/51493449-4d9a0d00-1dad-11e9-88fb-6e43c591786d.PNG">

So I think the proper fix would be to have the same behaviour as those apps and disambiguate the second set with a suffix.

**Comment by KeithMyers** [comment]
I think that is smart also to have consistency with the other monitoring programs.  I used SIV for years and beta tested for Ray Hinchcliffe many times.  He is usually the first to identify monitoring problems and develop support for the latest hardware that appears on the market.  A very smart man.

Just adding an identifier of the source to the reported values seems sufficient as that would show the reason for the different values and prevent further questions and comments.
